macOS-Mojave-libvirt.xml

```diff
@@ -0,0 +1,175 @@
+<domain type='kvm' xmlns:qemu='http://libvirt.org/schemas/domain/qemu/1.0'>
+<!--
+	macos Mojave libvirt XML configuration.
+
+        Run "virt-xml-validate macOS-Mojave-libvirt.xml" to validate this file.
+
+	You may hot-redirect USB with ease from GNOME Boxes and other goodies.
+
+	To install this file, you may place it at ~/.config/libvirt/qemu/
+	and run: virsh define macOS-Mojave-libvirt.xml.
+
+	This configuration has been tested in Fedora 27 with stock QEMU-KVM.
+
+	Move/rename images and loader/nvmram files and paths as you wish.
+
+	Don't forget to replace CHANGEME with your values.
+
+	Replace gnome-boxes path with libvirt path if you don't use it.
+
+	Adjust memory and currentMemory to 3145728 if you want only 3 GiB.
+
+	Consider removing some cpu features if your hardware lacks support.
+
+	Replace spice with vnc if you prefer it.
+
+	Current network configuration is a local bridge (192.157.12x.x).
+	Change it to if you prefer a public bridge instead:
+		Change interface to <interface type='user'>
+		and remove the <source bridge='virbr0'/>
+		Or use virt-manager to edit this line instead of virsh edit.
+
+	Note: Default configuration caused severe clock problems
+	under Fedora 27 w/ i7-5820K. This is because Darwin uses
+	tsc (time since last tick) for time, and for me did not
+	fall back to rtc in the event of a clock mismatch with
+	libvirt's default time source. Therefore we must explicitly
+	give the clock a tsc timer for kvm to pass to the guest.
+	See comments on the <kvm> and <clock> attributes.
+-->
+  <name>macos-mojave</name>
+  <uuid>2aca0dd6-cec9-4717-9ab2-0b7b13d111c3</uuid>
+  <title>macOS Mojave</title>
+  <memory unit='KiB'>4194304</memory>
+  <currentMemory unit='KiB'>4194304</currentMemory>
+  <vcpu placement='static'>4</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc-q35-2.11'>hvm</type>
+    <loader readonly='yes' type='pflash'>/home/CHANGEME/macOS/OVMF_CODE.fd</loader>
```

Comment:
It would be better to use a relative path instead of absolute path regarding OVMF

Reply:
libvirt seems only support absolute path.